test/core_tests.js

```diff
@@ -1,138 +1,156 @@
-    assert.equal( canvas.style.transformStyle,
-                  "preserve-3d",
-                  "canvas.style.transformStyle initialized correctly" );
-    assert.equal( canvas.style.transitionDelay,
-                  "0ms",
-                  "canvas.style.transitionDelay initialized correctly" );
-    // impress.js default values tries to set this to 1000ms, I'm completely confused about why that's not actually set in the browser?
-    assert.equal( canvas.style.transitionDuration,
-                  "0ms",
-                  "canvas.style.transitionDuration initialized correctly" );
-    assert.equal( canvas.style.transitionProperty,
-                  "all",
-                  "canvas.style.transitionProperty initialized correctly" );
-    assert.equal( canvas.style.transitionTimingFunction,
-                  "ease-in-out",
-                  "canvas.style.transitionTimingFunction initialized correctly" );
-                  
-    assert.equal( document.documentElement.style.height,
-                  "100%",
-                  "documentElement.style.height is 100%" );
-    
-    // Steps initialization
-    var step1 = document.querySelector( "div#step-1" );
-    assert.equal( step1.style.position,
-                  "absolute",
-                  "Step position is 'absolute'." );
-
-    assert.ok( step1.classList.contains("active"),
-               "Step 1 has active css class." );
-  }
-});
-
-// Note: Here we focus on testing the core functionality of moving between
-// steps, the css classes set and unset, and events triggered.
-// TODO: more complex animations and check position, transitions, delays, etc...
-// Those need to be separate html files, and there could be several of them.
-//
-// For now this just contains a single transition in order to get QUnit committed.
-// Complete test coverage needs to be added later.
-QUnit.test( "Impress Core API - Work in progress", function( assert ) {
-  // impress.js itself uses event listeners to manipulate most CSS classes. 
-  // Wait a short while before checking, to avoid race. 
-  // (See assertStepEnterWrapper and assertStepLeaveWrapper.)
-  var wait = 5; // milliseconds
-
-  var done = assert.async();
-  var step1 = document.querySelector( "div#step-1" );
-  var step2 = document.querySelector( "div#step-2" );
-  var root  = document.querySelector( "div#impress" );
-
-  // Things to check on impress:stepenter event -----------------------------//
-  var assertStepEnter = function( event ) {
-    assert.equal( event.target, step2,
-                  event.target.id + " triggered impress:stepenter event." );
-    assert.ok( step2.classList.contains("present"),
-               event.target.id + " set present css class." );
-    assert.ok( !step2.classList.contains("future"),
-               event.target.id + " unset future css class." );
-    assert.ok( !step2.classList.contains("past"),
-               event.target.id + " unset past css class." );
-    done();
+(function() {
```

Comment:
IIFE allows the declaration of private variables, it's a common pattern to reduce visibility in JS land.